signup-service/src/main/java/io/mosip/signup/helper/IdentityVerificationRequestValidator.java

```diff
@@ -0,0 +1,28 @@
+package io.mosip.signup.helper;
+
+import io.mosip.signup.api.dto.FrameDetail;
+import io.mosip.signup.dto.IdentityVerificationRequest;
+import io.mosip.signup.util.ErrorConstants;
+import org.apache.kafka.common.errors.InvalidRequestException;
```

Comment:
Why is the Exception class from the Kafka package used?  We can use SignupException and pass the correct error code to the exception constructor.